pgjdbc/src/main/java/org/postgresql/jdbc/PgStatement.java

```diff
@@ -222,15 +224,26 @@ public void handleResultRows(Query fromQuery, Field[] fields, List<Tuple> tuples
         @Nullable ResultCursor cursor) {
       try {
         ResultSet rs = PgStatement.this.createResultSet(fromQuery, fields, tuples, cursor);
-        append(new ResultWrapper(rs));
+        append(new ResultWrapper(rs, commandTypeOfQuery(fromQuery)));
       } catch (SQLException e) {
         handleError(e);
       }
     }
 
+    private @NonNull SqlCommandType commandTypeOfQuery(@Nullable Query query) {
```

Comment:
`Query fromQuery` is non-nullable, so why declare `Query query` as nullable here?

By convention we use, the types are non-nullable by default, so in 99% of the cases `@NonNull` should not be used: https://github.com/pgjdbc/pgjdbc/blob/master/CONTRIBUTING.md#null-safety

Reply:
Done